app/models/organisation.rb

```diff
@@ -16,7 +16,7 @@ def validate_in_register?
 
   def self.fetch_organisations_from_register
     UseCases::Organisation::FetchOrganisationRegister.new(
-      organisations_gateway: Gateways::OrganisationRegisterGateway.new
+      organisations_gateway: Gateways::OrganisationsGateway.new
```

Comment:
I know there have been some discussions around this.

I thought Gateways did know about where they get data from.
Like S3 gateway or a database gateway.

For me GovUkRegisterGateway would be more descriptive here.
@kjdchapman You have other ideas about this?

Reply:
I liked @emileswarts 's idea of getting `GovUkRegister` in the name, feel like it should mention `Organisations` too though. 

Without namespacing it looks pretty beefy - `GovUkRegisterOrganisationsGateway` :grimacing: